.github/ISSUE_TEMPLATE/github_user_access_request.md

```diff
@@ -22,6 +41,10 @@ If you are a member in bcgov or BCDevOps org and are inviting another user to th
 
 If you are __asking to invite yourself to an org__ please note that the request will **not** be automatically approved. An administrator of the bcgov and bcdevops github organizations will still need to approve the request. If this is not done in a timely manner please ping one of our Platform Admin @patrick.simonian @cailey.jones or @shelly.han in [#devops-operations](https://chat.developer.gov.bc.ca/channel/devops-operations) channel in Rocket.Chat.
```

Comment:
```suggestion
If you are __asking to invite yourself to an org__ please note that the request will **not** be automatically approved. The Platform Services Team will need to review the request to confirm your affiliation with the BC Government (**please indicate in the request if you are a government employee or include the name of the project and Ministry that you are working on, if you are a contractor, to expedite the approval process) before it can be approves. If this is not done in a timely manner please ping one of our Platform Admin @patrick.simonian @cailey.jones or @shelly.han in [#devops-operations](https://chat.developer.gov.bc.ca/channel/devops-operations) channel in Rocket.Chat.
```